spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/src/test/java/org/springframework/cloud/gcp/config/GcpConfigAutoConfigurationTest.java

```diff
@@ -54,13 +54,13 @@ public void testConfigurationValuesAreCorrectlyLoaded() {
 		loadEnvironment("spring.cloud.gcp.config.name=myapp",
 				"spring.cloud.gcp.config.profile=prod",
 				"spring.cloud.gcp.config.timeout=120000",
-				"spring.cloud.gcp.config.enabled=false",
+				"spring.cloud.gcp.config.enabled=true",
 				"spring.cloud.gcp.config.project-id=pariah");
 		GcpConfigProperties config = this.context.getBean(GcpConfigProperties.class);
 		assertEquals(config.getName(), "myapp");
 		assertEquals(config.getProfile(), "prod");
 		assertEquals(config.getTimeout(), 120000);
-		assertFalse(config.isEnabled());
+		assertTrue(config.isEnabled());
```

Comment:
Why this? `enabled` is `true` by default, testing if setting it to `false` works could make more sense.

Reply:
He set it to `true` above. So, the corresponding change to the assertion makes sense. If we want to test if setting to `false` works, that could be done in `disabledPropertySourceAvoidChecks` below.